Client/src/Validation/error.js

```diff
@@ -1,7 +1,7 @@
 const buildErrors = (prev, id, error) => {
 	const updatedErrors = { ...prev };
 	if (error) {
-		updatedErrors[id] = error.details[0].message?? "Validation error";
+		updatedErrors[id] = error.details[0].message ?? "Validation error";
 	} else {
 		delete updatedErrors[id];
 	}
```

Comment:
hasValidationErrors was refactored out when i worked on advanced settings page, it is not specific to any pages,
as it overlooking all pages for validation errors including white list unhandled fields to avoid unintended error being set such as below

```	if (
				![
					"clientHost",
					"refreshTokenSecret",
					"dbConnectionString",
					"refreshTokenTTL",
					"jwtTTL",
					"notify-email-list",
				].includes(err.path[0])
			) {
```

Reply:
Thanks for the explanation, I'll have to take a closer look at this function and how it is used.